src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -264,15 +270,48 @@ def set_consent_preferences(
                 raise HTTPException(
                     status_code=HTTP_400_BAD_REQUEST, detail=Pii(str(exc))
                 )
+    identity = Identity()
+    setattr(
+        identity,
+        provided_identity.field_name.value,  # type:ignore[attr-defined]
+        provided_identity.encrypted_value["value"],  # type:ignore[index]
+    )
```

Comment:
Pulling out the identity to feed this back into creating a PrivacyRequest.
Note that this creates a separate ProvidedIdentity for the PrivacyRequest. Not sure how I feel about this.

So both the ConsentRequest and PrivacyRequest are linked to different ProvidedIdentity records for this POC.